File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/acctype/pojo/AccType.java
package com.yss.ams.base.information.support.sys.acctype.pojo;

import com.yss.framework.api.common.co.BasePojo;

public class AccType extends BasePojo{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	/**
	 * 资产类型代码
	 */
	private String c_DAT_CODE = "";
	
	/**
	 * 资产类型名称
	 */
	private String c_DAT_NAME = "";
	
	/**
	 * 编号
	 */
	private int n_ORDER = 0;
	
	/**
	 * 资产类型代码父节点
	 */
	private String c_DAT_CODE_P = "";
	
	/**
	 * 资产类型（分类）
	 */
	private String c_DAT_TYPE = "";

	public String getC_DAT_CODE() {
		return c_DAT_CODE;
	}

	public void setC_DAT_CODE(String cDATCODE) {
		c_DAT_CODE = cDATCODE;
	}

	public String getC_DAT_NAME() {
		return c_DAT_NAME;
	}

	public void setC_DAT_NAME(String cDATNAME) {
		c_DAT_NAME = cDATNAME;
	}

	public int getN_ORDER() {
		return n_ORDER;
	}

	public void setN_ORDER(int nORDER) {
		n_ORDER = nORDER;
	}

	public String getC_DAT_CODE_P() {
		return c_DAT_CODE_P;
	}

	public void setC_DAT_CODE_P(String cDATCODEP) {
		c_DAT_CODE_P = cDATCODEP;
	}

	public String getC_DAT_TYPE() {
		return c_DAT_TYPE;
	}

	public void setC_DAT_TYPE(String cDATTYPE) {
		c_DAT_TYPE = cDATTYPE;
	}
}

File: AMS_基础信息组件/code/client/source/YssInformation.Support/Sys/Dictionary/Pojo/AccType.cs
using FAST.Common.Service.Pojo.Base;
using Newtonsoft.Json;

////namespace FAST.Platform.Dict.Pojo
////namespace YssInformation.Sys.Dictionary.Pojo
namespace YssInformation.Support.Sys.Dictionary.Pojo
{
    /// <summary>
    /// 资产类型pojo类
    /// </summary>
    public class AccType : BasePojo
    {
        /// <summary>
        /// 资产类型代码
        /// </summary>    
        private string c_DAT_CODE;

        /// <summary>
        /// 资产类型名称
        /// </summary>
        private string c_DAT_NAME;

        /// <summary>
        /// 编号
        /// </summary>
        private int n_ORDER;

        /// <summary>
        /// 资产类型代码P
        /// </summary>
        private string c_DAT_CODE_P;

        /// <summary>
        /// 资产类型（分类）
        /// </summary>
        private string c_DAT_TYPE;

        /// <summary>
        /// 属性：资产类型代码
        /// </summary>
        [JsonProperty(PropertyName = "c_DAT_CODE")]
        public string C_DAT_CODE
        {
            set { c_DAT_CODE = value; }

            get { return c_DAT_CODE; }
        }

        /// <summary>
        /// 属性：资产类型名称
        /// </summary>
        [JsonProperty(PropertyName = "c_DAT_NAME")]
        public string C_DAT_NAME
        {
            set { c_DAT_NAME = value; }

            get { return c_DAT_NAME; }
        }

        /// <summary>
        /// 属性：编号
        /// </summary>
        [JsonProperty(PropertyName = "n_ORDER")]
        public int N_ORDER
        {
            set { n_ORDER = value; }

            get { return n_ORDER; }
        }

        /// <summary>
        /// 属性：资产类型代码P
        /// </summary>
        [JsonProperty(PropertyName = "c_DAT_CODE_P")]
        public string C_DAT_CODE_P
        {
            set { c_DAT_CODE_P = value; }

            get { return c_DAT_CODE_P; }
        }


        /// <summary>
        /// 属性：资产类型代码P
        /// </summary>
        [JsonProperty(PropertyName = "c_DAT_TYPE")]
        public string C_DAT_TYPE
        {
            set { c_DAT_TYPE = value; }

            get { return c_DAT_TYPE; }
        }
    }
}
